OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -520,6 +520,10 @@ Action InitInputPanel(Widget panel)
 					{ "StopKey", "Stop" },
 					{ "ScatterKey", "Scatter" },
 					{ "StanceCycleKey", "Cycle Stance" },
+					{ "StanceHoldFireKey", "Hold fire" },
+					{ "StanceReturnFireKey", "Return fire" },
+					{ "StanceDefendKey", "Defend stance" },
```

Comment:
There isn't sufficient space to add 'stance' to all descriptions. When removed from all, will it be clear? Or should I change the layout so stance will take e.g. 2 columns?